github-projects@morgan-design.com/files/github-projects@morgan-design.com/applet.js

```diff
@@ -42,9 +42,9 @@ const Logger=imports.logger;
 const APPLET_ICON = global.userdatadir + "/applets/github-projects@morgan-design.com/icon.png";
 
 const NotificationMessages = {
-    AttemptingToLoad:   { title: "GitHub Explorer",					content: "Attempting to Load your GitHub Repos" },
-    SuccessfullyLoaded: { title: "GitHub Explorer",					content: "Successfully Loaded GitHub Repos for user ", append: "USER_NAME" },
-    ErrorOnLoad:		{ title: "ERROR:: GitHub Explorer ::ERROR", content: "Failed to load GitHub Repositories! Check applet Configuration" }
+    AttemptingToLoad:   { title: "GitHub Explorer",					content: _("Attempting to Load your GitHub Repos") },
+    SuccessfullyLoaded: { title: "GitHub Explorer",					content: _("Successfully Loaded GitHub Repos for user "), append: "USER_NAME" },
```

Comment:
No I don't say to remove the whitespace. I want it to remove from the translation template.
See my previous comment. The string in the brackets _() will be translated. And appending `+ " "` ensures that the whitespace is still there.

I like to not have trailing whitespaces in the translations. A lot of translators don't recognize, that there is a trailing white space or they ignore them. Then it looks weird in the translated language, when they do.